### ISSUE TITLE
Enhancement: Rename Actions/Stores

### ISSUE THREAD

**Issue by Antonio-Bennett** [ankurrsinghal/svelte-legos#53]
I'm of the opinion that actions should be renamed to drop the "Action" label from imports. This is because the `use:` syntax indicates that it is an action. Here is also the thread on Twitter discussing the possibility of renaming stores into readable and writable variants. https://twitter.com/ankurpsinghal/status/1637505767757455365?s=46&t=vXFY1pu_gVqC1tnNxYJ_Ow

Below are the different options versus currently.

```javascript
//Current
import { hoverAction } from "svelte-legos";

//suggested
import { hover } from "svelte-legos";
```
This gives us
```javascript
<div use:hoverAction on:hover={handler} />

//versus 

<div use:hover on:hover={handler} />
```

In terms of stores, appending `Store` isn't bad but as per the thread maybe explicitly stating readable/writable would be better.

The options are

```javascript
import { hoverStore } from "svelte-legos";

//versus

//This one seems to make the most sense to me
import { readableHover } from "svelte-legos";

//or maybe

import { hoverReadable } from "svelte-legos";

```

**Comment by symsmith** [comment]
I very much agree with this - see also #8 on the same topic

**Comment by Antonio-Bennett** [comment]
Yeah reading through that thread it would be nice if the imports are from either `svelte-legos/actions` or `svelte-legos/stores` that way it's even more explicit
